src/Rule/Callback.php

```diff
@@ -32,23 +37,29 @@ final class Callback implements
     use SkipOnErrorTrait;
     use WhenTrait;
 
+    /**
+     * @var object|null The object being validated.
+     */
     private ?object $objectValidated = null;
 
+    /**
+     * @param callable|null $callback Callable with the `function ($value, $rule, $context): Result` signature that
```

Comment:
Maybe add the note about `$callback` and `$method` about being mutually exclusive with each other?

Reply:
That's in the exception so the issue will be visible right away.

Reply:
I think having it mentioned in the docs additionally won't be redundant, like in `LimitTrait` for example.